src/Literature/LiteratureHelpers.tsx

```diff
@@ -11,6 +12,13 @@ export function showLiterature(fn: LiteratureName): void {
   for (const factionName of litObj.factionRumors) {
     Player.receiveRumor(factionName);
   }
-  const txt = `<i>${litObj.title}</i><br><br>${litObj.text}`;
+  const txt = (
+    <>
+      <i>{litObj.title}</i>
+      <br />
+      <br />
+      {litObj.text}
+    </>
+  );
   dialogBoxCreate(txt, true);
```

Comment:
```suggestion
  dialogBoxCreate(txt);
```
The second parameter should not be used. We only use it in 2 cases:
- Literature. We concatenate strings to create an HTML string, so we need HTML support in `dialogBoxCreate`.
- `ns.alert`. Legacy support.

Literature content is a React element now, so there is no need to call `dialogBoxCreate` with html=true.

Reply:
Done!

I only left it in due to my own uncertainty over what it controlled, and didn't try removing it, as it didn't produce any erroneous output.  Thanks for the pointer!